source/frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "frontend",
-  "version": "2.2.0-38.21",
```

Comment:
package-lock file needs also to be updated

Reply:
updated